src/main/java/software/amazon/kinesis/connectors/flink/internals/publisher/fanout/FanOutShardSubscriber.java

```diff
@@ -313,7 +318,7 @@ public void visit(SubscribeToShardEvent event) {
 		@Override
 		public void onError(Throwable throwable) {
 			LOG.debug("Error occurred on EFO subscription: {} - ({}).  {} ({})",
-				throwable.getClass().getName(), throwable.getMessage(), shardId, consumerArn);
+				throwable.getClass().getName(), throwable.getMessage(), shardId, consumerArn, throwable);
```

Comment:
How about:
```
if (LOG.isDebugEnabled()) {
	LOG.debug("Error occurred on EFO subscription: {} - ({}).  {} ({})",
			throwable.getClass().getName(), throwable.getMessage(), shardId, consumerArn, throwable);
} else {
	LOG.info("Error occurred on EFO subscription: {} - ({}).  {}",
			throwable.getClass().getName(), throwable.getMessage(), shardId, consumerArn);
}

```

Reply:
The error is already logged as a warning when processed by consumer in `handleError`. I left as debug here to stop printing the same errors multiple times